ci/test/pipeline.template.yml

```diff
@@ -103,6 +103,17 @@ steps:
     agents:
       queue: linux-x86_64
 
+  - id: check-merge-with-target
```

Comment:
Maybe not the best location between two docs jobs.

Reply:
Moved up and inserted after the two build steps.